### PR TITLE
Add IaaS-specific summaries?

### PR DESCRIPTION
**Pull request by dpb587-pivotal**
Content-wise, I was looking to have a place for CPI landing page to document their basic functionality (including new encryption info) and help bridge some of the core terms IaaSes deal with.

Specifically here...

 * provides entrypage and summary for basic requirements/support
 * easier links to cross-reference repository and support links
 * add encryption details for known iaases

I'm interested in feedback if this may be a useful direction or what could be improved, @mfine30 / @voelzmo / opinioners? If useful, I think next step would be to propose individual PRs of these changes and ask CPI leads to review/amend the content before merging and publishing.

### CODE REVIEW

**Comment by bingosummer** [comment]
I think, it's very useful. A question about Multi-CPI: which feature is required to support multi-cpi?
Azure CPI provides the method `info`. Is it enough?

File: content/azure.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+| BOSH              | Microsoft Azure |
+| ----------------- | --------------- |
+| Availability Zone | [Availability Zone](https://docs.microsoft.com/en-us/azure/availability-zones/az-overview) |
+| Virtual Machine   | [Virtual Machine](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes) |
+| Network Subnet    | [Virtual Network Subnet](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview) |
+| Virtual IP        | [Public IP](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-ip-addresses-overview-arm#public-ip-addresses) |
+| Persistent Disk   | [Disk Storage](https://azure.microsoft.com/en-us/services/storage/disks/) and [Managed Disks](https://azure.microsoft.com/en-us/services/managed-disks/) |
+| Disk Snapshot     | [Managed Disk Snapshot](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview#managed-disk-snapshots) |
+| Stemcell          |  |
```
Comment (mfine30) [inline content/azure.md]: Is the stemcell link intentionally missing?

File: content/aws.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+| Availability Zone | [Availability Zone](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html) |
+| Virtual Machine   | [EC2 Instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Instances.html) |
+| Network Subnet    | [VPC Subnet](https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html) |
+| Virtual IP        | [EC2 Elastic IP](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html) |
+| Persistent Disk   | [EC2 EBS Volume](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumes.html) |
+| Disk Snapshot     | [EC2 EBS Snapshot](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSSnapshots.html) |
+| Stemcell          | [EC2 Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) |
+
+
+## Feature Support
```
Comment (mfine30) [inline content/aws.md]: Is "Feature Support" referring to bosh features or IaaS features? Thoughts on adding a one-liner summary for this heading like we have under Concepts?

File: content/azure.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+
+### Network
+
+| Network Type | Support |
+| ------------ | ------- |
+| Manual       | Multiple networks per instance |
+| Dynamic      | Multiple networks per instance |
+| VIP          | Single network per instance |
+
```
Comment (mfine30) [inline content/azure.md]: Can we add an analogous sentence like we have for AWS? "The AWS CPI does not currently support multiple NICs being attached to a VM."

File: content/google.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+| BOSH              | Google Cloud Platform |
+| ----------------- | --------------------- |
+| Availability Zone | [Zone](https://cloud.google.com/compute/docs/regions-zones/) |
+| Virtual Machine   | [Virtual Machine Instance](https://cloud.google.com/compute/docs/instances/) |
+| Network Subnet    | [VPC Subnet](https://cloud.google.com/vpc/docs/vpc#vpc_networks_and_subnets) |
+| Virtual IP        | [Static External IP](https://cloud.google.com/compute/docs/ip-addresses/#reservedaddress) |
+| Persistent Disk   | [Persistent Disks](https://cloud.google.com/persistent-disk/) |
+| Disk Snapshot     | [Persistent Disk Snapshots](https://cloud.google.com/compute/docs/disks/create-snapshots) |
+| Stemcell          |  |
```
Comment (mfine30) [inline content/google.md]: link?

File: content/azure.md (hunk 1 above)
Comment (dpb587-pivotal) [reply content/azure.md]: Was lazy and hadn't finished filling a couple out. I'm not super familiar with some IaaSes, so will send separate PRs asking for individual CPI teams to finish+review in those cases.

File: content/aws.md (hunk 2 above)
Comment (dpb587-pivotal) [reply content/aws.md]: Good thought; will add a clarifying line.

File: content/azure.md (hunk 3 above)
Comment (dpb587-pivotal) [reply content/azure.md]: Will do

**Comment by dpb587-pivotal** [comment]
Pushed AWS/GCP/Azure and started a PR for OpenStack; can followup with a vSphere PR after more general feedback from Marco.

**Pull request closed without merging**